src/components/navigation/Navbar.scss

```diff
@@ -27,7 +27,7 @@
     height: 4px;
     margin-bottom: 5px;
     position: relative;  
-    background: $secondary;
+    background: white;
```

Comment:
Why have you decided not to use a variable?

Reply:
er, we didn't make a variable for white in the first place
would we do this as  something like $accent = white?